src/main/java/org/opentripplanner/standalone/config/routerconfig/updaters/VehiclePositionsUpdaterConfig.java

```diff
@@ -25,7 +26,31 @@ public static VehiclePositionsUpdaterParameters create(String updaterRef, NodeAd
       .since(V2_2)
       .summary("The URL of GTFS-RT protobuf HTTP resource to download the positions from.")
       .asUri();
+    var fuzzyTripMatching = c
+      .of("fuzzyTripMatching")
+      .since(V2_5)
+      .summary("Whether to match trips fuzzily.")
+      .asBoolean(false);
+    var features = c
+      .of("features")
+      .since(V2_5)
+      .summary("Which features of GTFS RT vehicle positions should be loaded into OTP.")
+      .asEnumSet(VehiclePositionFeature.class);
     var headers = HttpHeadersConfig.headers(c, V2_3);
-    return new VehiclePositionsUpdaterParameters(updaterRef, feedId, url, frequency, headers);
+    return new VehiclePositionsUpdaterParameters(
+      updaterRef,
+      feedId,
+      url,
+      frequency,
+      headers,
+      fuzzyTripMatching,
+      features
+    );
+  }
+
+  public enum VehiclePositionFeature {
+    POSITION,
+    STOP_POSITION,
+    OCCUPANCY,
   }
```

Comment:
@leonardehrenfried do you think this is the right set of values for enabling different features? I'll still have to add some handling to the GTFS API when position features are not enabled as it leads to npe currently.

Reply:
I think we can group position and stop position together, can't we?

Reply:
I had it like that at first but I think the position changes way more often than the "stop position", I think roughly the same refresh rate is enough for both occurance and stop position (although the stop position can change slightly more often).

Reply:
Ah, so in your case you're interested in occupancy and stop position?

Reply:
In our UI use case only about occupancy but I think I could also include the stop position information to OTP if some external user wants to use them through OTP.